lua/octo/pickers/telescope/entry_maker.lua

```diff
@@ -4,6 +4,23 @@ local utils = require "octo.utils"
 
 local M = {}
 
+---@param row table @A row from the issue or pull request list
+---@return string @The highlight group for the icon
+local function get_icon_highlight_group(row)
+  local highlight_group
+  if row.isDraft then
+    highlight_group = "OctoGrey"
+  elseif row.state == "OPEN" then
+    highlight_group = "OctoGreen"
+  elseif row.state == "CLOSED" then
+    highlight_group = "OctoRed"
+  elseif row.state == "MERGED" then
+    highlight_group = "OctoRed"
```

Comment:
Haven't looked at Octo src at all so double check the name but merged PRs are purple.

```suggestion
  elseif row.state == "MERGED" then
    highlight_group = "OctoPurple"
```

Reply:
Oops, didn't see your comments.